test/framework/resources/templates/vpntunnel-copy.yaml.tpl

```
@@ -0,0 +1,64 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: {{ .AppLabel }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .AppLabel }}
+  template:
+    metadata:
+      labels:
+        app: {{ .AppLabel }}
+    spec:
+      initContainers:
+      - image: alpine:3.10.3
```

```suggestion
      - image: eu.gcr.io/gardener-project/3rd/alpine:3.13.5
```
You could use the latest alpine from our gcr (instead of dockerhub) (but we also have `3.10.3` should you need that one specifically)

---

updated it